ssh-add.desktop

```diff
@@ -0,0 +1,4 @@
+[Desktop Entry]
+Name=ssh-add
+Exec=ssh-add
+Type=Application
```

Comment:
Good point, I didn't test. I also meant to point out in the README.md that you might want to specify the path to the particular key instead of default id_rsa, in the Exec..